extensions/authentication-jwt/src/__tests__/fixtures/controllers/user.controller.ts

```diff
@@ -95,4 +106,68 @@ export class UserController {
   async whoAmI(): Promise<string> {
     return this.user[securityId];
   }
+  // Routes using refreshtoken
+  @post('/users/refresh/login', {
+    responses: {
+      '200': {
+        description: 'Token',
+        content: {
+          'application/json': {
+            schema: {
+              type: 'object',
+              properties: {
+                accessToken: {
+                  type: 'string',
+                },
+                refreshToken: {
```

Comment:
if the function only returns `accessToken`, then `refreshToken` here should be removed IMO.

Reply:
Here both access token and refresh token would be sent back. So in spite of changing this IMO , we can have structure like:  
> on line 141 const token to accessToken

and
 
> on line 142 const accessToken to tokens

so that token would contain TokenObject(i.e both refresh Token and access Token)